R/rstudio_jobs.R

```diff
@@ -24,7 +24,8 @@ run_job <- function(text, tempfile_prefix = "file", jobname_prefix = "Job", ...)
 #' install_deps_job(create_args = list(direction = "all"))
 #' install_deps_job(dry_install = TRUE)
 #' }
-install_deps_job <- function(project = ".", project_type = "local", verbose = 1, create_args = list(), ...) {
+install_deps_job <- function(project = ".", project_type = "local", verbose = 1,
+                             create_args = list(renv_profile = Sys.getenv("RENV_PROFILE")), ...) {
```

Comment:
When running the jobs (and therefore the addins) we take the current active renv_profile (if there is one) - this is in contrast to when running the `dependency_table` function where the default is to use renv_profile = NULL - see dependency_table function